coinstacks/bitcoin/ingester/src/workers/address.ts

```diff
@@ -21,9 +28,9 @@ const onMessage = (worker: Worker) => async (message: Message) => {
     const tx = await blockbook.getTransaction(txid)
     msgLogger.trace({ blockHash: tx.blockHash, blockHeight: tx.blockHeight, txid: tx.txid }, 'Transaction')
 
-    const pTx = await parseTx(tx, address)
+    const pTx = await parser.parse(tx, address)
 
-    worker.sendMessage(new Message({ ...pTx, sequence, total } as SequencedBTCParseTx), client_id)
+    worker.sendMessage(new Message({ ...pTx, sequence, total } as SequencedTx), client_id)
```

Comment:
Why is "as" needed?

Reply:
I have addressed this before, but I am using as as a way to enforce the payload data being sent as json. If `SequencedTx` changes in any way, this will ensure we update.